servlet/src/main/scala/org/http4s/servlet/syntax/ServletContextSyntax.scala

```diff
@@ -14,17 +13,18 @@ trait ServletContextSyntax {
 
 final class ServletContextOps private[syntax] (val self: ServletContext) extends AnyVal {
 
-  /** Wraps an [[HttpRoutes]] and mounts it as an [[AsyncHttp4sServlet]] */
-  def mountService[F[_]: ConcurrentEffect](
+  /** Wraps an [[HttpRoutes]] and mounts it as an [[AsyncHttp4sServlet]]
+    *
+    * Assumes non-blocking servlet IO is available, and thus requires at least Servlet 3.1.
```

Comment:
The alternative is to require a blocking `ExecutionContext` that most users aren't going to need, or to provide an `Option[ServletIO]` which defaults to `None`, which is a pattern I never really liked much.

Reply:
How many people _want_ blocking servlets in today's world is a good question.

Reply:
Some of those managed platforms are stuck in antiquity. We've had a few requests. Though it has been a while.

It is also higher throughput, if you can afford a thread per request.